### ISSUE TITLE
Create cell with a background image

### ISSUE THREAD

**Issue by wrc-marek** [masaccio/numbers-parser#75]
Is there a way to create a cell with a background image?
I was trying this:

```python
from numbers_parser import Document, BackgroundImage

picture_path = "./test.jpg"
doc = Document("./template.numbers")

f = open(picture_path, mode="rb")
picture_data = f.read()
style = doc.add_style(
    name="Test",
    bg_image=BackgroundImage(picture_data, picture_path)
)

table = doc.sheets[0].tables[0]
table.write(0, 0, "Test", style=style)
table.set_cell_style(0, 1, style)

doc.save("new.numbers")
```

**Comment by masaccio** [comment]
Implemented in v4.9.0; publishing pending including an implementation of #74

**Comment by masaccio** [comment]
Now published. There's an example [in this test](https://github.com/masaccio/numbers-parser/blob/main/tests/test_styles.py#L422).

**Comment by wrc-marek** [comment]
Thanks, it's working!

The only problem that I see is that it's still have some problems with documents that contain already some background image.
I created manually document with image:

<img width="145" alt="Screenshot 2024-02-25 at 20 20 37" src="https://github.com/masaccio/numbers-parser/assets/98819454/1d5c7320-4416-45f9-8003-5f29d9ce37e4">

And after executing the script:

```
from numbers_parser import Document
doc = Document("./test.numbers")
doc.save("test2.numbers")
```

New file is missing a cat:
<img width="162" alt="Screenshot 2024-02-25 at 20 22 17" src="https://github.com/masaccio/numbers-parser/assets/98819454/5a9f649b-c6d1-48f3-b44a-43fd13277313">

**Comment by masaccio** [comment]
Is that cell otherwise empty?

**Comment by masaccio** [comment]
I am figuring yes (it failed when I tried). Fixed in v4.9.4.

**Comment by wrc-marek** [comment]
It's working, great job, thanks! 👍
